memory/src/main/java/org/opensearch/ml/memory/action/conversation/CreateInteractionRequest.java

```diff
@@ -171,6 +171,16 @@ public static CreateInteractionRequest fromRestRequest(RestRequest request) thro
             }
         }
 
+        boolean allFieldsEmpty = (input == null || input.trim().isEmpty())
+            && (prompt == null || prompt.trim().isEmpty())
+            && (response == null || response.trim().isEmpty())
+            && (origin == null || origin.trim().isEmpty())
+            && (addinf == null || addinf.isEmpty());
+        if (allFieldsEmpty) {
+            throw new IllegalArgumentException(
+                "At least one of the following parameters must be non-empty: " + "input, prompt_template, response, origin, additional_info"
```

Comment:
Just curious to know if I just give addition info, will this feature work? Did you test?

Reply:
Yes, I tested it manually and I also have a corresponding [unit test](https://github.com/opensearch-project/ml-commons/pull/3283/files#:~:text=public%20void%20testFromRestRequest_WithAllFieldsEmpty_Fails,%7D).